.rubocop_todo.yml

```diff
@@ -1,26 +1,31 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2019-04-01 11:37:39 -0400 using RuboCop version 0.65.0.
+# on 2019-04-01 16:38:34 -0400 using RuboCop version 0.65.0.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
-# Offense count: 11
+# Offense count: 16
 Metrics/AbcSize:
   Max: 29
 
-# Offense count: 11
+# Offense count: 12
 # Configuration parameters: CountComments, ExcludedMethods.
 # ExcludedMethods: refine
 Metrics/BlockLength:
   Max: 204
 
-# Offense count: 3
+# Offense count: 2
+# Configuration parameters: CountComments.
+Metrics/ClassLength:
+  Max: 140
+
+# Offense count: 4
 Metrics/CyclomaticComplexity:
-  Max: 7
+  Max: 11
```

Comment:
:(

Reply:
I could not come up with a easy refactor, it could use another set of eyes :)

Reply:
Let's make sure to circle back to this one. I don't have a refactor for this one except for pushing every if statement embedded in whens into their own methods.